repo.go

```diff
@@ -1 +1,41 @@
-package repo
+package main
+
+import (
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"text/template"
+
+	"github.com/gorilla/mux"
+	"github.com/russross/blackfriday"
+)
+
+type content struct {
+	Body string
+}
+
+func generateHTML() {
+	// Update repo
+	exec.Command("git", "checkout", "-f").Output()
+	exec.Command("git", "pull").Output()
+
+	input, _ := ioutil.ReadFile("./README.md")
+	body := string(blackfriday.MarkdownCommon(input))
```

Comment:
You should sanitize the output from blackfriday, just in case someone sneaks a malicious piece of user content and we accidentally merge it.

See https://github.com/russross/blackfriday#sanitize-untrusted-content.